src/_scss/components/_footerLinkToAdvancedSearch.scss

```diff
@@ -26,7 +26,7 @@
                 font-weight: $font-semibold;
             }
         }
-        .agency-search-button {
+        .footerLinkToAdvancedSearch-search-button {
```

Comment:
Not too familiar with [BEM](http://getbem.com/naming/), but shouldn't this be `.footerLinkToAdvancedSearch__search-button`? Or, just `.footerLinkToAdvancedSearch__button`? I guess I'm confused as to when we should be using `-` versus `__` and when we should be using `camel case` for scss.

Reply:
made it this footerLinkToAdvancedSearch__button